doc/exporter-guide.md

```diff
@@ -0,0 +1,49 @@
+# Exporter Developer Guide
+
+An exporter sends traces and metrics to any backend that is capable of consuming them. The exporter itself can change without requiring a change in your client code.
+
+OpenTelemetry exporters can be contributed by anyone, and we provide support for several open source backends and vendors out-of-the-box, example Zipkin, Jaeger, Prometheus. This document will provide a means for developers to create their own. We strongly recommended to create a dedicated package for newly added exporter, example: `@opentelemetry/exporter-myexporter`.
```

Comment:
> We strongly recommended to create a dedicated package for newly added exporter, example: `@opentelemetry/exporter-myexporter`.

I would remove this sentence. It implies that any user can create a package under the `@opentelemetry` namespace. If a user is creating their own exporter outside of our official packages (vendor specific, some custom functionality) then we do not care what their name is. If they are creating one in our repo, then the package name is part of the process from the start and will follow conventions we have already established.

Also a couple minor grammar tweaks

```suggestion
We provide support for several open source backends and vendors out-of-the-box like Zipkin, Jaeger, and Prometheus, but OpenTelemetry exporters follow a public interface can be implemented by anyone. This document describes the process for developers to create their own exporter if the provided ones do not meet their needs.
```